recipes/x13as/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -ex
+
+# in contrast to the usual build orchestrators (which pass LDFLAGS to the
+# compiler), the hand-spun upstream build script passes them directly to the
+# linker, so strip the `-Wl,...` bits that are meant to tell the compiler to
+# forward to the linker (but which make no sense for the linker itself)
+export LDFLAGS="$(echo $LDFLAGS | sed 's/-Wl,//g')"
+# also replace `-Wl,rpath,...` formulation; the linker expects separation
+# by space, i.e. `-rpath[,-link] ...`
+export LDFLAGS="$(echo $LDFLAGS | sed 's/-rpath,/-rpath /g')"
+export LDFLAGS="$(echo $LDFLAGS | sed 's/-rpath-link,/-rpath-link /g')"
+
+if [[ "$target_platform" == linux-* ]]; then
+    # where libquadmath is found in our setup
+    export LDFLAGS="$LDFLAGS -L$CONDA_BUILD_SYSROOT/../lib"
+    # needs to explicitly link glibc & libm
+    export LDFLAGS="$LDFLAGS -L$CONDA_BUILD_SYSROOT/lib64 -lc -lm"
+    # also needs compiler runtime
+    export LDFLAGS="$LDFLAGS -lgcc_s"
+else
+    export LDFLAGS="$LDFLAGS -framework CoreFoundation"
+fi
+# both platforms need to link libgfortran
+export LDFLAGS="$LDFLAGS -lgfortran"
```

Comment:
@isuruf, I was wondering if you'd have an idea here why `libgfortran` isn't found on osx, resp. `libgcc_s` on linux. After a bit of necessary massaging, I'm using `FFLAGS` and `LDFLAGS` from the fortran compiler activation, but for some reason `$PREFIX/lib` doesn't seem to be picked up correctly.

It's a bit weird that this is all so manual, but I guess that's because there's no real build orchestrator at play here, and it's just a hand-written upstream makefile. Still, it's surprising that a fortran compiler needs help to find libgfortran. 🙄 

Note: in contrast to autotools or at least CMake, `LDFLAGS` isn't being passed to the compiler, but to the linker itself; hence we need to strip `-Wl,` etc.

It _could_ maybe have something to do with having to use an older gfortran...? Doesn't sound very plausible, but I'm a bit out of ideas. Unfortunately, anything newer than gfortran 10 (inclusive) fails to compile the sources...